site/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "site",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "docusaurus": "docusaurus",
+    "start": "docusaurus start",
+    "build": "docusaurus build",
+    "swizzle": "docusaurus swizzle",
+    "deploy": "docusaurus deploy",
+    "clear": "docusaurus clear",
```

Comment:
I thought this was a typo at first... But no... Docusaurus really uses `clear` instead of `clean`...